packages/core/src/auth/providers/credentialsProviderManager.ts

```diff
@@ -23,13 +26,18 @@ export class CredentialsProviderManager {
     private readonly providerFactories: CredentialsProviderFactory[] = []
     private readonly providers: CredentialsProvider[] = []
 
+    @withTelemetryContext({ name: 'getAllCredentialsProvider', class: credentialsProviderManagerClassName, emit: true })
```

Comment:
And this emits every call, possible chance this will be noisy. But it should hopefully not break anything before we remove it